rest-fights/pom.xml

```diff
@@ -239,6 +246,7 @@
           <systemPropertyVariables>
             <java.util.logging.manager>org.jboss.logmanager.LogManager</java.util.logging.manager>
             <maven.home>${maven.home}</maven.home>
+            <pact_do_not_track>true</pact_do_not_track>
```

Comment:
@holly-cummins I _think_ this is the right thing to do? We don't want to enable https://docs.pact.io/telemetry for the superheroes (or anyone cloning/running superheroes locally)?